python/lsst/validate/drp/check.py

```diff
@@ -21,8 +21,10 @@
 from __future__ import print_function, division
 
 import numpy as np
+from scipy.optimize import curve_fit
```

Comment:
I'm not a Stack developer, but do you need to add `scipy` to `ups/validate_drp.table` now? Same for `matplotlib` actually.